File: code-coaches/medium/symbols/symbols.py
def remove_symbols(encoded: str) -> str:
    # symbols: a char that is not alphanumeric or a space
    return "".join([char for char in encoded if char.isalnum() or char.isspace()])


def main():
    encoded = input()
    print(remove_symbols(encoded))


if __name__ == "__main__":
    main()

File: code-coaches/medium/symbols/symbols.cpp
#include <iostream>

std::string remove_symbols(std::string encoded)
{
    // symbols: a char that is not alphanumeric or a space
    std::string decoded;
	for (auto &ch : encoded)
	{
		if (isspace(ch) || isalnum(ch))
			decoded += ch;
	}

	return decoded;
}

int main()
{
	std::string encoded;
	getline(std::cin, encoded);

	std::cout << remove_symbols(encoded);

	return 0;
}
